src/flamegraph.js

```diff
@@ -674,8 +672,19 @@ export default function () {
   chart.label = chart.setLabelHandler
 
   chart.search = function (term) {
+    let searchFunction
+    if (typeof term === 'function') {
```

Comment:
Can cause confusion with users.

Reply:
Thanks for taking a look at the PR.

As you point out, the function argument can be either a literal or a function. In my view, this would be natural to users, because so many other d3 functions have this behavior. As one example:

https://github.com/d3/d3-selection#selection_text

    If the value is a constant, then all elements are given 
    the same inner HTML; otherwise, if the value is a function, 
    it is evaluated for each selected element, ...

So, I hope you'll consider merging this as-is.